boards/wio_lite_mg126/src/lib.rs

```diff
@@ -83,7 +83,7 @@ define_pins!(
     pin d24 = b11,
 
     /// Digital 22: MISO
-    pin d22 = b12,
+    //pin d22 = b12,
```

Comment:
Is this something you accidentally included while testing?

Reply:
Good catch. This is something I forgot about. PB12 doesn't exist on D21G variants. I just looked up the schematic, and it looks like this must have been a typo. It should be PA12. I'll correct it and re-push.